backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -1233,4 +1233,12 @@ class TestOperator extends VeloxWholeStageTransformerSuite {
       checkOperatorMatch[HashAggregateExecTransformer]
     }
   }
+
+  test("Support ApproximatePercentile") {
+    runQueryAndCompare("""
+                         |SELECT approx_percentile(col, array(0.5, 0.4, 0.1), 100)
```

Comment:
I remember Spark has  `percentile_approx`. Is that one supported by this patch either?